packages/client/src/Config.ts

```diff
@@ -76,12 +76,12 @@ export interface ControlLayerConfig {
     peerDescriptor?: NetworkPeerDescriptor
 
     /**
-     * The port to use for the client's Network Node WebSocket server.
-     * If not specified, the server will not be started.
+     * The port range used to find a free port for the client's network layer WebSocket server.
+     * If not specified, a server will not be started.
      * The server is used by the network layer to accept incoming connections
      * over the public internet to improve the network node's connectivity.
      */
-    webSocketPort?: number
+    webSocketPortRange?: PortRange
```

Comment:
The lowercase naming convention is usually used in the public API (e.g in `NetworkPeerDescriptor` and in Broker plugin name).

```suggestion
    websocketPortRange?: PortRange
```